config.dev.toml

```diff
@@ -43,6 +43,14 @@ LogLevel = "debug"
 # Env override: ATHENS_CLOUD_RUNTIME
 CloudRuntime = "none"
 
+# ActivateDebugHandler specifies if the pprof endpoints should be exposed.
+# Note that this option is not meant to be activated forever on a server
+# and should be desactivated once not needed.
+ActivateDebugHandler = false
```

Comment:
What do you think about calling this `EnablePprof` ? I think most Go developers would immediately know what this means since `pprof` is part of the standard tooling. Debug, on the other hand could mean a lot of things, such as verbose debug logging.

Reply:
Yes, you are right, it will be clearer :) 
I was thinking that it might be relavant to activate other tools as well but if needed later, will change the config name.